pkg/sloop/webserver/webserver.go

```diff
@@ -40,6 +40,7 @@ import (
 const (
 	debugViewKeyTemplateFile      = "debugviewkey.html"
 	debugListKeysTemplateFile     = "debuglistkeys.html"
+	debugHistogramFile            = "debughistogram.html"
 	debugConfigTemplateFile       = "debugconfig.html"
 	debugTemplateFile             = "debug.html"
```

Comment:
can you update debug.html to include a link to this new page?

Reply:
Yes its already updated https://github.com/salesforce/sloop/pull/109/files#diff-39bf5ff9e161f28bf9b37e5848d9d739